build-scripts/patches/0000-Kubelite-integration.patch

```diff
@@ -89,8 +89,8 @@ index 59662e425bb..c3e4408685a 100644
  
  			// make the kubelet's config safe for logging
  			config := kubeletServer.KubeletConfiguration.DeepCopy()
-@@ -290,7 +295,7 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
- 			klog.V(5).InfoS("KubeletConfiguration", "configuration", kubeletServer.KubeletConfiguration)
+@@ -298,7 +303,7 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
+ 			klog.V(5).InfoS("KubeletConfiguration", "configuration", config)
  
```

Comment:
I suppose this is because of https://github.com/canonical/microk8s/commit/89061be829e02415200ce16dd43e809b7b743027?

Reply:
Right, rebasing to address this.